test/sql/window/test_basic_window.test

```diff
@@ -73,6 +73,48 @@ SELECT empno, first_value(empno) OVER (PARTITION BY depname ORDER BY empno) fv F
 3	1
 4	1
 
+# last_value
+query III
+SELECT depname, empno,
+	last_value(empno) OVER (
+		PARTITION BY depname ORDER BY empno ASC
+		ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING
+		) fv
+FROM empsalary
+ORDER BY 1, 2
+----
+develop	7	11
+develop	8	11
+develop	9	11
+develop	10	11
+develop	11	11
+personnel	2	5
+personnel	5	5
+sales	1	4
+sales	3	4
+sales	4	4
+
+# nth_value
+query III
+SELECT depname, empno,
+	nth_value(empno, 2) OVER (
```

Comment:
Could you add tests:
* Where either of the the parameters is a constant NULL
* Where either of the parameters is a column that contains NULL values
* Where the second parameter (offset) is not a constant
* Where the first parameter is a constant

Reply:
At some point these should be done for all window functions. Breaking up `test_basic_window` seems like the way to go.